src/openrct2/windows/guest_list.c

```diff
@@ -141,7 +141,7 @@ static sint32 _window_guest_list_highlighted_index; // 0x00F1EE10
 static sint32 _window_guest_list_selected_tab;      // 0x00F1EE12
 static sint32 _window_guest_list_selected_filter;   // 0x00F1EE06
 static sint32 _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32 _window_guest_list_selected_view;  // 0x00F1EE13
+static uint32 _window_guest_list_selected_view;		// 0x00F1EE13
```

Comment:
What do you mean exactly? I used tabs on them all, VS2017 seems to be a little odd with tabs. What I mean is tabs seems to be able to be different sizes, and I never use spaces. For the hell of it I redid all of the tabs and Git says I made no changes.